source/developers/projects/craftercms/index.rst

```diff
@@ -1057,15 +1057,20 @@ Useful Git Commands
 
 Here are some useful Git commands for setting up our Crafter CMS project.
 
+  .. note::
+
+     You may notice a few ``.keep`` files in your repository.  Those ``.keep`` files are automatically generated by Studio when empty folders are created, since Git doesn't keep track of folders (and Studio does). It's best if you just leave them there and don't add them to ``.gitignore``
+
+
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 Copy Crafter CMS repository and clone submodules
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
     .. code-block:: bash
 
-       git clone https://github.com/craftercms/craftercms.git
-       cd craftercms
-       git submodule clone
+       ➜ git clone https://github.com/craftercms/craftercms.git
```

Comment:
Are you sure about this indentation (seems that it's in tabs). Will it render correctly?

Reply:
Hi!
I wanted to highlight that the commands are entered one at a time so I copied my terminal prompt...
It looks okay after I built the docs... Here's a screenshot:

<img width="996" alt="Screen Shot 2021-02-09 at 11 01 03 AM" src="https://user-images.githubusercontent.com/25483966/107390877-23b40000-6ac6-11eb-9340-f64617a975b9.png">

Then I realized if the user does a copy and paste of the commands, it might not work because of the prompts added... Will revert to original. Thank you.